canonical-web-and-design/snapcraft.io
# Docs front end

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1051

## QA
- Ensure you have the search key available and set in your environment (or pass it to `./run` - see below)
- Pull the branch
- `./run --env SEARCH_API_KEY={the-api-key}`
- Visit http://0.0.0.0:8004/docs
- Use the docs
- Read the docs
- Become 1 with the docs
- Use the search
- Paginate through the search results
- Click on things
- Listen to your heart
